### ISSUE TITLE
Suggestion: Create a command which allows people without the 'Manage Server' permission to change a server's region.

### ISSUE THREAD

**Issue by brybrant** [jonas747/yagpdb#730]
Skip to TLDR.

Recently, many people in all Discord servers have been experiencing problems with connecting to voice channels. During a raid of 50+ people, half the people in a voice channel get connection issues like "no route" and "RTC connecting" for no apparent reason. The only way to resolve this problem seems to be by changing the server's region. However, I cannot be there for every raid from start to finish to fix the issue by changing the server's region in case this happens because I have other things to do. I would give my moderators the ability to manage the server, but this also gives them permission to mess with the server's YAGPDB settings in the website dashboard which could be disastrous. I would like to give my moderators the ability to switch the server's region by typing a command instead of giving them 'Manage Server' permission.

TLDR:
I found this github repo of a bot which can change a server's region by typing commands: https://github.com/Decicus/discord-server-change

I would love you forever if you added this as a native YAGPDB bot command! ❤️ thanks
